tests_e2e/test_suites/images.yml

```diff
@@ -46,6 +46,30 @@ image-sets:
       - "ubuntu_1804"
       - "ubuntu_2004"
 
+   # agent persist firewall service not running on flatcar distro since agent can't install custom service due to read only filesystem.
+   # so creating a separate image set for agent-persist-firewall scenario
```

Comment:
should we instead add a property similar to 'skip_on_clouds', but to skip distros. otherwise we need to keep in sync all those endorsed image sets

Reply:
ideally yes. I'll take this as separate work item